src/libs/sfdk/virtualmachine.cpp

```diff
@@ -571,19 +574,20 @@ void VirtualMachine::refreshConfiguration(const QObject *context, const Functor<
     });
 }
 
-void VirtualMachine::refreshState(Sfdk::VirtualMachine::Synchronization synchronization)
+void VirtualMachine::refreshState(const QObject *context, const Functor<bool> &functor)
```

Comment:
Something to think about (definitely not fix now!): We seem to have this pattern of passing these (context, functor) pairs around a lot. Wouldn't it be convenient if we had a type which contained both?

Reply:
I also thought about, but on caller side it would be less convenient - it would require an extra pair of braces - while the profit on implementation side would be just subtle I think.